src/test/java/org/mybatis/typehandlers/threetenextra/DayOfMonthTypeHandlerTest.java

```diff
@@ -0,0 +1,93 @@
+/**
+ *    Copyright 2016-2017 the original author or authors.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package org.mybatis.typehandlers.threetenextra;
+
+import java.time.DateTimeException;
+import org.apache.ibatis.type.TypeHandler;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import org.junit.Test;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import org.threeten.extra.DayOfMonth;
+
+/**
+ * @author Björn Raupach
+ */
+public class DayOfMonthTypeHandlerTest extends BaseTypeHandlerTest {
+
+    private static final TypeHandler<DayOfMonth> TYPE_HANDLER = new DayOfMonthTypeHandler();
+    private static final DayOfMonth INSTANT = DayOfMonth.now();
```

Comment:
I copied and pasted most of the test cases. I agree the naming of the variable is not adequate.